imageregistry/v1/types.go

```diff
@@ -200,6 +200,12 @@ type ImageRegistryConfigStorageS3 struct {
 	// Optional, defaults based on the Region that is provided.
 	// +optional
 	RegionEndpoint string `json:"regionEndpoint,omitempty"`
+	// chunkSizeMiB defines the size of the multipart upload chunks of the S3 API.
+	// The S3 API requires multipart upload chunks to be at least 5MB.
```

Comment:
I left a comment before about what we should express when the value is unset, that should be added here